Automattic/wp-calypso
# Remove un-used focused launch code

#### Proposed Changes

Depends on merging these first:
- https://github.com/Automattic/wp-calypso/pull/71049
- https://github.com/Automattic/wp-calypso/pull/71047

Removes all code related to "focused launch" functionality which opened an entirely new launch flow directly from the editor. The feature has been un-used for a while now:

- Editor toolkit "focused launch" parts
- `@automattic/launch` package

There could be some parts that can be re-used here regarding launch flow starting from the launchpad, so worth reviewing the code a bit closer!

Meanwhile, none of this is used so it should go.

There's a bit of code here that can probably be removed as well — I'll clean that up as part of "Gutenboarding" cleanup:

https://github.com/Automattic/wp-calypso/blob/c7646c50a74a61d60d95d7e847dc45580dfe648a/client/gutenberg/editor/calypsoify-iframe.tsx#L419-L427

#### Testing Instructions

* Everything should continue working as-usual; editor, site editor, launch flow, etc.
* Make sure to [apply ETK plugin changes to your sandbox](https://github.com/Automattic/wp-calypso/pull/71051#issuecomment-1346212818) when testing the editor

#### Pre-merge Checklist





Related to #
